docs/ui/insight/tools/cli.md

```diff
@@ -0,0 +1,16 @@
+---
+title: CLI
+linkTitle: CLI
+description: Interact with your database in RedisInsight using CLI
+weight: 10
+
+---
+
+The CLI is accessible at any time within the application. It includes:
```

Comment:
While RedisInsight provides a comprehensive graphical interface for managing Redis or Redis Stack, there are many advanced operations that you can perform only by a command-line interface. For such cases, you can use RedisInsight CLI which supplements the graphical interface and provides you greater flexibility and control over your database by running Redis commands.

RedisInsight CLI is accessible at any time within your database, allowing you to run Redis commands while investigating the list of keys in Browser, receiving or publishing messages in Pub/Sub, troubleshooting performance issues, or optimizing the performance and memory usage of your database based on recommendations displayed in Database Analysis.

It includes a type-ahead help to assist with the syntax of Redis commands. Also, RedisInsight CLI is integrated with Command Helper which allows you to read about your Redis command used in CLI, or filter and search for any Redis command to learn more about its purpose and syntax.